SwiftShell.playground/Contents.swift

```diff
@@ -8,4 +8,10 @@
 
 import SwiftShell
 
-echo("Hello, SwiftShell!")
```

Comment:
Exclude changes to SwiftShell.playground. This file should keep as it is and be changed by actual users.